interface/patient_file/summary/add_edit_issue.php

```diff
@@ -731,15 +733,15 @@ function divclick(cb, divid) {
                     </div>
                 </div>
                 <div class="form-group">
-                    <label class="col-form-label col-2" for="form_begin"><?php echo xlt('Begin Date'); ?>:</label>
+                    <label class="col-form-label col-4" for="form_begin"><?php echo xlt('Begin Date'); ?>:</label>
```

Comment:
This is actually incorrect. The Bootstrap 4 grid system goes all the way up to col-12, not col-14.